web/js/containers/timeline/timeline.js

```diff
@@ -99,10 +100,21 @@ class Timeline extends React.Component {
       rangeSelectorMax: {
         end: false, start: false, startOffset: -50, width: 50000,
       },
+      matchingTimelineCoverage: {},
+      isDataCoveragePanelOpen: false,
+      shouldIncludeHiddenLayers: false,
     };
+
+    const {
+      changeDate,
+      onUpdateStartDate,
+      onUpdateEndDate,
+      onUpdateStartAndEndDate,
+    } = this.props;
+
     // left/right arrows
-    const throttleSettings = { leading: true, trailing: false };
-    this.debounceDateUpdate = lodashDebounce(this.props.changeDate, 8);
+    const throttleSettings = throttleSettings;
```

Comment:
Noticed this when resolving merge conflicts with this rebased onto my current lint changes branch... 

Looks like we're assigning a new `const` to its `undefined` self here?  Not sure if that was intentional or not.